java-bootcamp-bahia-t1/src/MusicPlayer/MusicPlayerState.java

```diff
@@ -0,0 +1,7 @@
+package MusicPlayer;
+
+public interface MusicPlayerState {

```

Comment:
State pattern , to encapsulate varying behavior for the same routine based on an MusicPlayer's state object.